idl/idl-parser/src/org/jetbrains/dukat/idlParser/util.kt

```diff
@@ -0,0 +1,14 @@
+package org.jetbrains.dukat.idlParser
+
+import org.antlr.v4.runtime.ParserRuleContext
+import org.antlr.v4.runtime.tree.ParseTree
+import org.antlr.v4.runtime.tree.TerminalNode
+import org.antlr.webidl.WebIDLLexer
+
+internal fun List<ParseTree>?.filterIdentifiers(): List<ParseTree> = this?.filter {
```

Comment:
I'm very against of having any file named util we need this.
let's name it whatever we find appropriate but not utils.

Also I suggest to do following - I know there are some places in code where we are extending Lists but that's honestly a very dubious practice  - let's have an agreement not to extend any generic entity but rather pass it as a param.